CHANGES.rst

```diff
@@ -71,11 +71,22 @@ Bugs
 ++++
 
 # ``0`` with a given precision (like in ```0`3```) is now parsed as ``0``, an integer number.
+#. ``Random[]`` works now.
 #. ``RandomSample`` with one list argument now returns a random ordering of the list items. Previously it would return just one item.
 #. Origin placement corrected on ``ListPlot`` and ``LinePlot``.
 #. Fix long-standing bugs in Image handling
 #. Units and Quantities were sometimes failing. Also they were omitted from documentation.
 
+PyPI Package requirements
++++++++++++++++++++++++++
+
+Mathics3 now aims at a more richer set of functionality.
+
+Therefore NumPy and Pillow are required Python packages where they had
+been optional before.  In truth, probably running Mathics without one
+or both probably did not work well if it worked at all; we had not
+been testing setups that did not have NumPy.
```

Comment:
I guess NumPy is a requirement for SymPy

Reply:
https://libraries.io/pypi/sympy/1.10.1/tree says that only mpmath is required.  However looking at the code I see lots of numpy imports for a lot of functions. I wonder if this too is not tested that well. (Or if so not very usable without NumPy). 

scikit-image requires numpy. 
pillow  like sympy doesn't per se, but probably is not very usable without numpy. 

If Mathics3 was working well  without NumPy, NumPy detection was done cleanly, and the code was working fast without it then this wouldn't be an issue. But currently we need the speed and functionality of Numpy and what it provides to Pillow, SymPy, and Scikit-image.  In the future I am hoping that our large numeric data and multi-dimensional arrays will get stored there as well since numpy is performant while just about anything else isn't.

And NumPy is a well-established package in the Python ecosystem.